src/components/EditableField/EditableField.Input.tsx

```diff
@@ -175,15 +176,19 @@ export class EditableFieldInput extends React.Component<InputProps> {
   /* istanbul ignore next */
   handleKeyDown = event => {
     const isEnter = event.key === key.ENTER
-    const isEscape = event.key === key.ESCAPE
     const isDropdownTrigger = event.target.classList.contains(
       OTHERCOMPONENTS_CLASSNAMES.dropdownTrigger
     )
```

Comment:
I am curious on why the Escape part was removed?

Reply:
Before this input was only allowing `enter` and `escape` key events to bubble up to the parent. In the case of enter, it was only bubbling up when no in a dropdown trigger. With the exception of the dropdown case, we now want all key events to bubble up.